src/base/bittorrent/torrenthandle.cpp

```diff
@@ -521,15 +521,20 @@ int TorrentHandle::piecesHave() const
 
 qreal TorrentHandle::progress() const
```

Comment:
I think it's useful to have two getters, one for current operation progress, and the second for download progress. Some kind of remote clients haven't store state.